includes/Core/Admin/Screens.php

```diff
@@ -264,7 +264,7 @@ private function no_access_redirect_dashboard_to_splash() {
 		}
 
 		// Redirect to splash screen if user is allowed to authenticate.
```

Comment:
This comment is a bit stale now as the condition isn't specific to authentication anymore but also probably redundant as the code is clear enough on its own IMO 🙂 

```suggestion
```